string weights_name
Tensor2d weights
string bias_name
Tensor1d bias